snapcraft/tests/repo/test_deb.py

```diff
@@ -265,31 +265,24 @@ def setUp(self):
     def get_installable_packages(self, packages, target_arch=''):
         installable = []
         for pkg in packages:
-            if not packages[pkg].installed:
```

Comment:
I'm not sure I understand why you removed this check. If a package is already installed, it shouldn't be marked as installable.